src/attributes.jl

```diff
@@ -488,6 +488,11 @@ struct ConstraintDualStart <: AbstractConstraintAttribute end
 
 The assignment to the constraint primal values in result `N`.
 If `N` is omitted, it is 1 by default.
+
+Given a constraint `function-in-set`, the `ConstraintPrimal` is the value of the
+function evaluated at the primal solution of the variables. For example, given
+the constraint `2x + y <= 1`, and a primal solution of `(x,y) = (1,2)`, the
```

Comment:
There are multiple ways to encode this constraint in MOI, could help to be more precise (ScalarAffineFunction).